### PR TITLE
agda 2.6.4

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

- [x] `resource` blocks have been checked for updates.

### CODE REVIEW

**Comment by p-linnane** [comment]
```
==> cabal --store-dir=/opt/homebrew/Cellar/agda/2.6.4/libexec v2-install --jobs=4 --max-backjumps=100000 --install-method=copy --installdir=/opt/homebrew/Cellar/agda/2.6.4/lib/agda
  Error: cabal: Could not resolve dependencies:
  [__0] trying: agda-stdlib-utils-1.7.2 (user goal)
  [__1] next goal: base (dependency of agda-stdlib-utils)
  [__1] rejecting: base-4.18.0.0/installed-4.18.0.0 (conflict: agda-stdlib-utils
  => base>=4.9.0.0 && <4.18)
  [__1] skipping: base-4.18.1.0, base-4.18.0.0 (has the same characteristics
  that caused the previous version to fail: excluded by constraint '>=4.9.0.0 &&
  <4.18' from 'agda-stdlib-utils')
  [__1] rejecting: base-4.17.2.0, base-4.17.1.0, base-4.17.0.0, base-4.16.4.0,
  base-4.16.3.0, base-4.16.2.0, base-4.16.1.0, base-4.16.0.0, base-4.15.1.0,
  base-4.15.0.0, base-4.14.3.0, base-4.14.2.0, base-4.14.1.0, base-4.14.0.0,
  base-4.13.0.0, base-4.12.0.0, base-4.11.1.0, base-4.11.0.0, base-4.10.1.0,
  base-4.10.0.0, base-4.9.1.0, base-4.9.0.0, base-4.8.2.0, base-4.8.1.0,
  base-4.8.0.0, base-4.7.0.2, base-4.7.0.1, base-4.7.0.0, base-4.6.0.1,
  base-4.6.0.0, base-4.5.1.0, base-4.5.0.0, base-4.4.1.0, base-4.4.0.0,
  base-4.3.1.0, base-4.3.0.0, base-4.2.0.2, base-4.2.0.1, base-4.2.0.0,
  base-4.1.0.0, base-4.0.0.0, base-3.0.3.2, base-3.0.3.1 (constraint from
  non-upgradeable package requires installed instance)
  [__1] fail (backjumping, conflict set: agda-stdlib-utils, base)
  After searching the rest of the dependency tree exhaustively, these were the
  goals I've had most trouble fulfilling: base, agda-stdlib-utils
```

**Comment by iMichka** [comment]
I think we still need one part of the patch, let's try this.

**Comment by iMichka** [comment]
Let's try this by bumping agda-stdlib-utils.cabal too.

**Comment by shhyou** [comment]
Stdlib 1.7.3 that fixes large-indices issues to work with Agda 2.6.4 was released!
https://github.com/agda/agda-stdlib/releases/tag/v1.7.3

File: Formula/a/agda.rb [hunk 1]
```diff
@@ -70,7 +61,7 @@ def install
       system "cabal", "v2-update"
       system "cabal", "--store-dir=#{libexec}", "v2-install", *cabal_args, "--installdir=#{lib}/agda"
       system "./GenerateEverything"
-      system bin/"agda", "-i", ".", "-i", "src", "--html", "--vim", "README.agda"
+      system bin/"agda", "--large-indices", "-i", ".", "-i", "src", "--html", "--vim", "README.agda"
```
Comment (shhyou) [inline Formula/a/agda.rb:64]: With the newly released stdlib 1.7.3, there would be no need for `--large-indices`.

**Review by SMillerDev — APPROVED** [review]

**Pull request merged**